### PR TITLE
In the generic CMake class: always be verbose

### PR DESCRIPTION
**Pull request by wpoely86**

### CODE REVIEW

File: easybuild/easyblocks/generic/cmakemake.py [hunk 1]
```diff
@@ -96,6 +96,9 @@ def configure_step(self, srcdir=None, builddir=None):
             if value is not None:
                 options.append("-D%s='%s'" % (option, value))
 
+        # show what CMake is doing by default
+        options.append("-DCMAKE_VERBOSE_MAKEFILE=ON")
```
Comment (boegel) [inline easybuild/easyblocks/generic/cmakemake.py]: maybe only set this if debug logging is enabled?

Comment (wpoely86) [inline easybuild/easyblocks/generic/cmakemake.py]: Why? The logging options already decide over that `run_cmd` should display. And I cannot think of a single reason why we wouldn't want the full output?

Comment (boegel) [inline easybuild/easyblocks/generic/cmakemake.py]: well, bigger logs ;)

but you're right, I see no reason not to include this

others have been doing this for ages, cfr. https://bugzilla.redhat.com/show_bug.cgi?id=474053

**Comment by boegel** [comment]
going in, thanks @wpoely86!

**Pull request merged**